modules/oe_webtools_captcha/tests/modules/oe_webtools_captcha_mock/src/Controller/MockController.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_webtools_captcha_mock\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\State\State;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\HttpFoundation\JsonResponse;
+
+/**
+ * Controller to mock the captcha validation endpoint.
+ */
+class MockController extends ControllerBase {
+
+  /**
+   * The state service.
+   *
+   * @var \Drupal\Core\State\State
+   */
+  protected $state;
+
+  /**
+   * Constructs a new MockController instance.
+   *
+   * @param \Drupal\Core\State\State $state
+   *   The state service.
+   */
+  public function __construct(State $state) {
+    $this->state = $state;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container) {
+    return new static(
+      $container->get('state'),
+    );
+  }
+
+  /**
+   * Mock validation.
+   *
+   * @return \Symfony\Component\HttpFoundation\JsonResponse
+   *   A JSON response.
+   */
+  public function validate(): JsonResponse {
```

Comment:
If we are mocking the validation we don't test the Webtools response but a hardcoded one from this controller.
Checking the [Webtools captcha wiki](https://webgate.ec.europa.eu/fpfis/wikis/pages/viewpage.action?spaceKey=webtools&title=Sliding+Captcha) there are 2 parameters `wt_captcha_answer` and `wt_captcha_sid` that should be used in order to validate the response. Can we do something similar to `MockWcloudMiddleware` and assert all the cases instead of hardcoded responses?

Reply:
Checked with @hernani, testing the response is handled should be enough. No need for testing the actual Webtools response.